modules/account-lib/src/coin/hbar/ifaces.ts

```diff
@@ -1,18 +1,14 @@
 import { Ed25519PublicKey, Ed25519PrivateKey } from '@hashgraph/sdk';
+import { TransactionBody } from '@hashgraph/sdk/lib/generated/TransactionBody_pb';
+import { SignatureList, SignatureMap } from '@hashgraph/sdk/lib/generated/BasicTypes_pb';
 import { KeyPair } from '.';
 
 export interface TxData {
-  id: string;
+  sigs?: SignatureList.AsObject;
+  sigmap?: SignatureMap.AsObject;
+  body?: TransactionBody.AsObject;
```

Comment:
is there a case where this could be undefined? this is forcing you to use `!` when you want to access a value from it

Reply:
I think that it shouldn't be undefined, the sigs and sigmap could be undefined so those are ok.

Reply:
I think you're right. I've changed body field to be mandatory.